src/lib/transport.js

```diff
@@ -37,17 +40,24 @@ function createPacketAndCondition ({
   const address = destinationAccount + '.~' + protocol + '.' + receiverId +
     (id ? ('.' + id) : '')
 
-  const blobData = omitUndefined({
-    expires_at: expiresAt,
-    data: data
-  })
+  const details = createDetails({
+    unsafeHeaders: Object.assign({
+      'Key-Id': receiverId, // TODO: is this right?
```

Comment:
I think I prefer one of these:

``` http
Key: hmac-sha-256 W2WuqXqCedW3hj4UsQdzBg
```

Here are some examples from HTTP:

``` http
Authorization: Basic QWxhZGRpbjpvcGVuIHNlc2FtZQ==
Content-Type: text/html; charset=utf-8
Content-Disposition: attachment; filename="fname.ext"
Cookie: $Version=1; Skin=new;
Forwarded: for=192.0.2.60;proto=http;by=203.0.113.43
```

Authorization seems like the closest analog and it's a header a lot of developers have dealt with.

[Non-standard](http://progrium.com/blog/2012/12/17/http-signatures-with-content-hmac/), but when I saw it I thought it was very elegant and simple:

``` http
Content-HMAC: <hash mechanism> <base64 encoded binary HMAC>
```

Example:
```
Content-HMAC: sha1 f1wOnLLwcTexwCSRCNXEAKPDm+U=
```

Saves 18 bytes, still easy to parse, still easy enough to guess that hmac-sha-256 is some kind of type and the base64 thingy is the actual key. Can't really come up with a good reason not to use this kind of format.